### DISCUSSION TITLE
Stack Overflowとも連携できるようにして欲しい

### DISCUSSION THREAD

**Discussion by lamrongol** [lapras-inc/lapras-user-community#18] [Ideas]
技術力を測るにはGitHubやZennも有効ですが、Stack Overflowでの評価も重要だと思います。Stack Overflowは英語版やメタなどで評価が別々に集計されますが、以下のように一つのプロフィールページから他のSOでの評価・リンクも表示されるので日本語版とだけでも連携できれば大丈夫だと思います。
ユーザー: Loran - スタック・オーバーフロー https://ja.stackoverflow.com/users/816/loran?tab=profile

**Comment by UserSuccessTeamLAPRAS** [comment]
LAPRASサポートです。
ご意見ありがとうございます！改善のご要望として開発チームに共有させていただきました。

今後ともLAPRASをよろしくお願いします。

**Comment by lamrongol** [comment]
QuoraやMondはITエンジニア専門のQ&Aサイトでは無いですが、IT関連の質問も多く、また、普通のQ&Aサイトと違って回答者の経歴などがわかるようになっていて一般に質が高いと思います。また、現在でもnoteのような必ずしも技術力と関係ないサイトとも連携できるので、QuoraやMondとも連携できるようにして頂きたいです。

**Reply by UserSuccessTeamLAPRAS** [reply]
LAPRASユーザーサポートです。
ご意見ありがとうございます！改善のご要望として開発チームに共有させていただきました。

今後ともLAPRASをよろしくお願いします。
